Automattic/newspack-electionkit
# refactor: remove google maps api key from version control

### All Submissions:






### Changes proposed in this Pull Request:



Removes Google Maps API key from version control, and replaces it with a `wp-config.php` constant. A possible future improvement will be to create a Settings screen for the plugin and allow the Google Maps API to be input there.

### How to test the changes in this Pull Request:

1. Switch to this branch, view post/page with the shortcode on it. Input a zip and submit. Observe an error is returned. The error on the page is generic, although the JSON returned should indicate the cause to be a missing Google API key.
2. Add the key to `wp-config.php` as described in the README.
3. Repeat the test, verify the plugin works correctly.

### Other information:
